lib/msf/core/payload/linux.rb

```diff
@@ -95,6 +95,13 @@ def initialize(info = {})
             ""
           ]
         ),
+        Msf::OptBool.new('MeterpreterBackground',
+          [
+            true,
+            'Fork meterpreter to background as a daemon',
+            false
+          ]
+        )
```

Comment:
So I looked into how this is [implemented](https://github.com/rapid7/mettle/blob/master/mettle/src/service.c) in Mettle. It looks like if `background` is set and there is no persistence, then it forks itself and calls [`setsid(2)`](https://linux.die.net/man/2/setsid). This is basically the [same thing](https://github.com/rapid7/metasploit-payloads/blob/a4dc937eff96b1193eddcb498c5f02f787e85511/python/meterpreter/meterpreter.py#L1631-L1636) the Python Meterpreter does when the `MeterpreterTryToFork` option is set.

I think it would make more sense therefore to rename this option to `MeterpreterTryToFork` so it's consistent across the Meterpreter types that implement this functionality rather than call it one thing for Linux/Mettle and another thing for Python.

Completely off-topic for this PR, but in general I'd like to move towards feature and option parity where possible between the Meterpreters.

Reply:
Absolutely, I'll rename this option. I was not aware of the Python Meterpreter equivalent, thanks for the heads up!